src/com/untamedears/realisticbiomes/GrowthConfig.java

```diff
@@ -320,17 +330,23 @@ public double getRate(Block block) {
 	
 	double sunlightChecks(Block block) {
 		double rate = 1.0;
-		int sunlightIntensity;
-		if (block.getType().isTransparent()) {
-			sunlightIntensity = block.getLightFromSky();
-		} else {
-			sunlightIntensity = block.getRelative(BlockFace.UP).getLightFromSky();
-		}
-		// apply multiplier if the sunlight is not at maximum
-		if (sunlightIntensity < MAX_LIGHT_INTENSITY) {
+		Block up = block.getRelative(BlockFace.UP);
+		int sunIntensity = up.getLightFromSky();
+		if(ignoreForLight == null) {
 			rate *= notFullSunlightMultiplier;

```

Comment:
`notFullSunlightMultiplier ` this is now always applied! should only be applied if not at full sunlight.